docker/Dockerfile

```diff
@@ -28,8 +32,11 @@ ARG BASE_IMAGE=openjdk:8-jdk
 
 FROM maven:3.6-jdk-8 as builder
 
+ARG CI
 ARG MVN_ARG
 
+ENV CI ${CI}
```

Comment:
what’s this？

Reply:
Both GitHub Action and Travis will add `CI` into env.

https://github.com/apache/incubator-kyuubi/blob/master/build/mvn#L27-L29

Reply:
I think the name is not readable, can we change it to a more meaning name ? or can we use `MAVEN_CLI_OPTS` directly ?

Reply:
How about change to
```
ARG LESS_MAVEN_LOG
ENV CI ${LESS_MAVEN_LOG}
```

Reply:
The previous PR title and description may be misleading. Updated and please review again.